diceRolled.txt

```diff
@@ -0,0 +1,8 @@
+### This file displays the outcome of die after rolling.
+###The value after the semicolon is the new rolled amount.	 
+	 
+loaded:20;11	numbered:10;6	fudge:6;1	fudge:10;1	fudge:20;1	 
+loaded:4;4	numbered:10;10	fudge:6;-1	numbered:100;97	numbered:5;2	 
+fudge:8;0	numbered:8;6	loaded:8;8	 
+numbered:10;4	numbered:10;7	numbered:10;2	numbered:10;7	 
+fudge:6;-1	loaded:6;5	loaded:6;4	loaded:6;4	 
```

Comment:
@srtinkey yes, I was trying to add that in but I couldn't figure out how to do it. When printing the objects to the file, I passed each item in the loop using the basic die object which doesn't have a .getLoaded() method because it's only needed for the loaded die. Without the method in that interface, I couldn't call it in this loop. I wasn't sure how to go about that. Should I add the .getLoaded() method in the interface and then to the other die classes with a null return so that I'm able to call it for the loaded dice objects? Or is there a better way to go about this? Maybe @ninjapanzer could provide some insight on this as well lol